save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/TestExecutionController.kt

```diff
@@ -100,6 +101,9 @@ class TestExecutionController(
                     }
             }
         }
+        .map { testExecution ->
+            testExecution.extended()
```

Comment:
maybe `toExtended()` or `extend()`? `extended()` might mean to return true (like in cpp)

Reply:
Renamed to `toExtended()` in cf3d3a453d455beeb84b178cf78b4e6d55fd8184.